packages/snaps-cli/src/commands/manifest/manifest.ts

```diff
@@ -28,13 +89,37 @@ const steps: Steps<ManifestContext> = [
     },
   },
   {
-    name: 'Validating the snap manifest.',
-    task: async ({ input, options, spinner }) => {
+    name: 'Evaluating the Snap bundle.',
+    condition: ({ config }) => config.evaluate,
```

Comment:
Just confirming: Eval can be turned off with a CLI argument right?

Reply:
Added an `--eval` option, so it can be disabled with `--no-eval`. It still defaults to the value provided in the config if not set.